src/main/java/com/patchworkmc/annotation/AnnotationProcessor.java

```diff
@@ -49,25 +66,42 @@ public AnnotationVisitor visitAnnotation(String descriptor, boolean visible) {
 			// Ignore Kotlin metadata
 
 			return super.visitAnnotation(descriptor, visible);
-		} else {
-			Patchwork.LOGGER.warn("Unknown class annotation: " + descriptor + " " + visible);
-			return new AnnotationPrinter(super.visitAnnotation(descriptor, visible));
 		}
+
+		if (isForgeAnnotation(descriptor)) {
+			Patchwork.LOGGER.warn("Unknown Forge Annotation " + descriptor);
+		}
+
+		annotationStorage.acceptClassAnnotation(descriptor, className);
+		return new AnnotationPrinter(super.visitAnnotation(descriptor, visible));
```

Comment:
I think it would make sense to only print the annotation if it's a forge annotation?